shared/common-adapters/avatar.desktop.js

```diff
@@ -139,7 +139,7 @@ class Avatar extends Component<void, Props, State> {
         {this.props.backgroundColor &&
           <div
             style={{...avatarStyle,
-              ...borderStyle,
```

Comment:
Looks like this removes the logic that handles a `this.props.borderColor` passed in to the component. I don't see it being used anywhere, so you can probably delete the `const borderStyle = this.props.borderColor ? {borderRadius...` line too.